roles/builder/meta/main.yml

```diff
@@ -1,5 +1,7 @@
+---
 galaxy_info:
   standalone: false
   description: >
     setup osbuild image build server
-dependencies: []
+dependencies:
+  - osbuild.composer.setup_server
```

Comment:
Adding this dependency seems to force the setup_server role to run before running the image_server role even when running manually.

Reply:
It does indeed. I'll fix.

Reply:
@matoval after looking at this, it's correct to have the requirement as the role is currently written because if you try to run the `builder` role on a fresh system that has never run the `setup_server` role before then it will fail because the first task modifies state of the `cockpit.socket` systemd unit but never attempts to install the package first. Also, the second task sets group permissions for the user to the `weldr` group which also doesn't exist on a system that hasn't run the `setup_server` role before. We likely need to refactor the `builder` role considerably so we can drop the dependency, but for now it's needed.

Reply:
@maxamillion Currently the builder role doesn't run as root but the setup_server role does. Running the setup_server as a dependency of the builder role fails because it isn't running as root.

Reply:
If you attempted to run the builder role against a system that never had the setup_server role run against it, then the builder role would fail also if not run as root because an unprivileged user can't modify the systemd unit state nor the group membership.